Utils/azure-toolkit-ide-libs/azure-toolkit-ide-common-lib/src/main/java/com/microsoft/azure/toolkit/ide/common/experiment/ExperimentationService.java

```diff
@@ -59,7 +59,7 @@ public ExperimentationService create() {
     private void updateFeatures() {
         try {
             final Response response = client.newCall(request).execute();
```

Comment:
May use try with resource here to ensure response is closed, refers https://github.com/square/okhttp/blob/master/okhttp/src/jvmMain/kotlin/okhttp3/Call.kt#L51